src/utils/constants.cxx

```diff
@@ -163,4 +163,19 @@ DEFAULT_CONST(socket_listener_backlog, 1);
 DEFAULT_CONST(socket_listener_stack_size, 1000);
 /// Allow up to five sockets to be pending for accept() in SocketListener.
 DEFAULT_CONST(socket_listener_backlog, 5);
-#endif
\ No newline at end of file
+#endif
+
+/// Default stack size to use for OpenMRN tasks on the Arduino platform.
+DEFAULT_CONST(arduino_openmrn_stack_size, 4096);
+
+#ifdef ESP32
+#include <esp_task.h>
+/// Default thread priority for any OpenMRN owned tasks on the ESP32 platform.
+/// Note: This is set to one priority level lower than the TCP/IP task uses on
+/// the ESP32.
+DEFAULT_CONST_DEFERRED(arduino_openmrn_task_priority, ESP_TASK_TCPIP_PRIO - 1);
+#else
+/// Default thread priority for any OpenMRN owned tasks on the Arduino
+/// platform.
+DEFAULT_CONST(arduino_openmrn_task_priority, 0xdefa01);
```

Comment:
I'm not sure this will work for you. You pass it to xTaskCreate. In OpenMRN we have os_task_create which interprets this constant and sets it to the default (middle) priority level.

Reply:
Should we default this to -1 instead? I copied this as-is from OpenMRNLite.h.